docs/shared/recipes/tips-n-tricks/switch-to-workspaces-project-references.md

```diff
@@ -0,0 +1,230 @@
+# Switch to Workspaces and Project References
+
+In order to take advantage of the [performance benefits](/concepts/typescript-project-linking#typescript-project-references-performance-benefits) of TypeScript project references, you need to use package manager workspaces for local [project linking](/concepts/typescript-project-linking). If you are currently using TypeScript path aliases for project linking, follow the steps in this guide to switch to workspaces project linking and enable TypeScript project references.
```

Comment:
As mentioned in a previous comment, package manager workspaces are not a requirement for using TS project references. However, they are recommended regardless of whether you use TS project references. They enable native Node.js module resolution rather than having to resolve through a custom TypeScript resolution for TS path aliases.